Packs/CommonPlaybooks/Playbooks/playbook-URL_Enrichment_-_Generic_v2_README.md

```diff
@@ -40,9 +39,9 @@ This playbook does not use any sub-playbooks.
 | URL.Data | The enriched URL. | string |
 | DBotScore | The DBotScore object. | unknown |
 | URL.Malicious | Whether the detected URL was malicious. | unknown |
-| URL.Vendor | The vendor that labeled the URL as malicious. | string |
-| URL.Description | The additional information for the URL. | string |
+| URL.Vendor | Vendor that labeled the URL as malicious. | string |
+| URL.Description | Additional information for the URL. | string |
 
 ## Playbook Image
 ---
-![URL_Enrichment_Generic_v2](https://raw.githubusercontent.com/demisto/content/1bdd5229392bd86f0cc58265a24df23ee3f7e662/docs/images/playbooks/URL_Enrichment_Generic_v2.png)
+![URL Enrichment - Generic v2](../doc_files/URL_Enrichment_-_Generic_v2.png)
```

Comment:
@MLainer1  shouldnt your validation for relative urls fail on this?

Reply:
I think it did :) see here the build on the earlier commit - https://app.circleci.com/pipelines/github/demisto/content/206781/workflows/19fd72b1-a561-47cb-9b00-fe735b1210ac/jobs/597577